mystery_o_matic/clues.py

```
@@ -504,6 +643,36 @@ def string_spanish(self):
+
+        if r == 0:
+            s += "Увидел(а) "
+        elif r == 1:
+            s += "Заметил(а) "
+        elif r == 2:
+            s += "Разглядел(а) "
+        else:
+            raise ValueError("Invalid random number: " + str(r))
+
+        if not self.object_is_alive:
+            return f'{self.subject}: "Я был(а) потрясён(а), увидев тело {self.object} в {self.place}_LOC в {self.time}"'
```

Ruff (0.15.5)

[warning] String contains ambiguous `а` (CYRILLIC SMALL LETTER A). Did you mean `a` (LATIN SMALL LETTER A)?

(RUF001)